flycheck.el

```diff
@@ -8152,6 +8152,21 @@ See URL `https://eslint.org' for more information about ESLint."
           (let-alist (caar (flycheck-parse-json output))
             .messages)))
 
+(defun flycheck-eslint--find-working-directory (_checker)
+  "Look for a working directory to run CHECKER in.
+This will either be the directory that contains `.eslintrc' by supports
+configuration files formats or `.eslintignore', if no such file is found in
+the directory hierarchy, searches `node_modules' directory to detect root of project."
```

Comment:
Please rewrap :)

Reply:
I don't understand

Reply:
All docstrings looks similar for me, you need a space between first and second line?

Reply:
Yes, and try not to make documentation lines longer than 60 chars.

Reply:
I think `M-x fill-paragraph` with the cursor inside the docstring should do it, right?

Reply:
Was that also `M-q`?

Reply:
@Simplify On a stock Emacs, yes. But I favor suggesting calling commands directly, as not everyone has the same keybindings, and so that can be confusing ;)

Reply:
Done